app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(input_str: str) -> None:
+    old_name, new_name = input_str.strip("cp ").split()
+    if old_name != new_name:
+        with (open(old_name, "r") as input_file,
+              open(new_name, "a") as output_file):
```

Comment:
The file is opened in append mode ('a') instead of write mode ('w'). According to the task description, the function should copy the whole content to the new file, which implies overwriting any existing content.